src/js/preshuffle.s

```diff
@@ -1009,6 +1038,61 @@ Do16BitSubtractionForEXP:
   .byte $00
 .popseg
 
+;;; ----------------------------------------------------
+; [in] y - Offset for the current enemy we are displaying health for
+.reloc
+UpdateEnemyHPDisplay:
+  ; We scratch x so save it before running
+  txa
+  pha
+    lda ObjectHP, y
+    tax
+    lda ObjectDef, y
+    ; mask off all but the lowest bit (since the enemy HP is only 9bits right now)
+    and #$01
+    cpx RecentEnemyCurrHPLo
+    bne +
+    cmp RecentEnemyCurrHPHi
+    bne +
+    beq CheckMax ; unconditional
++   ; A = enemy curr hp hi, x = enemy curr hp lo
+    sta RecentEnemyCurrHPHi
+    txa
+    sta RecentEnemyCurrHPLo
+    tya
+    pha
+      lda #DISPLAY_NUMBER_ENEMYHP
+      jsr DisplayNumberInternal
+    pla
+    tay
+CheckMax:
```

Comment:
Can you make this a local label?  (I think it's an @-prefix, but I don't remember exactly)